app/src/main/java/ru/netology/fmhandroid/dao/ClaimDao.kt

```diff
@@ -0,0 +1,54 @@
+package ru.netology.fmhandroid.dao
+
+import androidx.room.*
+import kotlinx.coroutines.flow.Flow
+import ru.netology.fmhandroid.dto.Claim
+import ru.netology.fmhandroid.dto.ClaimCommentWithCreator
+import ru.netology.fmhandroid.dto.FullClaim
+import ru.netology.fmhandroid.entity.ClaimCommentEntity
+import ru.netology.fmhandroid.entity.ClaimEntity
+
+@Dao
+interface ClaimDao {
+    // Сортируй тут по датам дурачок :)
+    @Query(
+        "SELECT * FROM ClaimEntity ORDER BY planExecuteDate ASC, createDate DESC"
+    )
+    fun getAllClaims(): Flow<List<FullClaim>>
+
+    @Query("SELECT * FROM ClaimEntity WHERE status LIKE :firstStatus OR status LIKE :secondStatus ORDER BY planExecuteDate ASC, createDate DESC")
+    fun getClaimsOpenAndInProgressStatuses(
+        firstStatus: Claim.Status,
+        secondStatus: Claim.Status
+    ): Flow<List<FullClaim>>

```

Comment:
Так не универсальнее будет?
```suggestion
    @Query("SELECT * FROM ClaimEntity WHERE status IN :statuses ORDER BY planExecuteDate ASC, createDate DESC")
    fun getClaimsInStatuses(statuses: List<Claim.Status>): Flow<List<FullClaim>>
```